autoload/SpaceVim/layers/default.vim

```diff
@@ -122,7 +122,7 @@ function! SpaceVim#layers#default#config() abort
   call SpaceVim#mapping#space#def('nnoremap', ['b', 'd'], 'call SpaceVim#mapping#close_current_buffer()', 'kill-this-buffer', 1)
   call SpaceVim#mapping#space#def('nnoremap', ['b', 'D'],
         \ 'call SpaceVim#mapping#kill_visible_buffer_choosewin()',
-        \ 'kill-this-buffer', 1)
+        \ 'kill-this-buffer with vimchoosewin', 1)
```

Comment:
I think keep same format will be better, `kill-the-buffer-with-vimchoosewin`? or just use `kill-the-select-window`, maybe we will not use vimchoosewin in the feature.

Reply:
How about the `kill the buffer by selecting`? This expression sounds more clear

Reply:
nice, your english is better than me. :)